packages/frontend/src/components/campaigns/campaign/chain/index.tsx

```diff
@@ -0,0 +1,15 @@
+"use client";
+
+import type { SupportedChain } from "@metrom-xyz/contracts";
+import styles from "./styles.module.css";
```

Comment:
Styles as last import.

Reply:
Oh right, addressed in 29e717d56185787009a3d86f3a1966ed97f8c112